### ISSUE TITLE
MPRIS PlayPause broken

### ISSUE THREAD

**Issue by georgefst** [Spotifyd/spotifyd#940]
Version 0.3.0 on a RPi. This is a recent regression, though I'm unfortunately unsure what version I was running previously when this worked.

I've been using the command `dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotifyd /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.PlayPause` to pause and resume playback, but it now only seems to work for resume, and not pause.

This is a log from when the command is sent to a playing instance of `spotifyd`:
<details><summary>Click to show logs</summary>

```
<!-- PLEASE PASTE YOUR LOGS BELOW THIS LINE WHEN REPORTING BUGS. Make sure to run spotifyd using the `--verbose` flag -->
event Readable Token(50331652)
D-Bus i/o poll ready: 23 is Ready(Readable)
D-Bus i/o unix ready: 23 is Readable | Aio
loop process - 1 events, 0.000s
scheduling Read for: 4
event Readable Token(50331652)
loop process - 1 events, 0.000s
handle_msgs: (MethodCall, Some("/org/mpris/MediaPlayer2"), Some("org.mpris.MediaPlayer2.Player"), Some("PlayPause"))
loop poll - 3.94092717s
loop time - Instant { tv_sec: 3884434, tv_nsec: 504586052 }
loop process, 280.498µs
Polling message stream
msgstream found Ok(Ready(Some((MethodCall, Some("/org/mpris/MediaPlayer2"), Some("org.mpris.MediaPlayer2.Player"), Some("PlayPause")))))
Polling message stream
msgstream found Ok(NotReady)
Sending status to server: [kPlayStatusPlay]
D-Bus i/o poll ready: 23 is Ready(Readable)
D-Bus i/o unix ready: 23 is Readable | Aio
scheduling Read for: 4
scheduling Read for: 5
flushing framed transport
writing; remaining=771
framed transport flushed
loop poll - 724.05µs
loop time - Instant { tv_sec: 3884434, tv_nsec: 506793423 }
loop process, 372.034µs
Polling message stream
msgstream found Ok(NotReady)
event Readable | Writable Token(46137349)
loop process - 1 events, 0.000s
attempting to decode a frame
frame decoded from buffer
attempting to decode a frame
scheduling Read for: 5
flushing framed transport
framed transport flushed
loop poll - 143.316867ms
loop time - Instant { tv_sec: 3884434, tv_nsec: 651242781 }
loop process, 63.463µs
Polling message stream
msgstream found Ok(NotReady)
loop poll - 15.629µs
loop time - Instant { tv_sec: 3884434, tv_nsec: 651478020 }
loop process, 59.463µs
event Readable | Writable Token(46137349)
attempting to decode a frame
frame decoded from buffer
loop process - 1 events, 0.000s
attempting to decode a frame
scheduling Read for: 5
flushing framed transport
framed transport flushed
loop poll - 2.175973243s
loop time - Instant { tv_sec: 3884436, tv_nsec: 827540429 }
loop process, 62.148µs
Polling message stream
msgstream found Ok(NotReady)
kMessageTypePause "FP3" 91913ebe9d0641f2e6f331b3650be6ee2e279cc3 1980850649 1619036034582 kPlayStatusPlay
Sending status to server: [kPlayStatusPause]
scheduling Read for: 5
flushing framed transport
writing; remaining=772
framed transport flushed
loop poll - 248.738µs
loop time - Instant { tv_sec: 3884436, tv_nsec: 828372737 }
loop process, 58.241µs
command=Pause
== Stopping sink ==
event Readable | Writable Token(46137349)
loop process - 1 events, 0.000s
attempting to decode a frame
frame decoded from buffer
attempting to decode a frame
scheduling Read for: 5
flushing framed transport
framed transport flushed
loop poll - 32.11826ms
loop time - Instant { tv_sec: 3884436, tv_nsec: 860580423 }
loop process, 60.944µs
Polling message stream
msgstream found Ok(NotReady)
loop poll - 12.204µs
loop time - Instant { tv_sec: 3884436, tv_nsec: 860799106 }
loop process, 59.018µs
loop poll - 601.902617ms
loop time - Instant { tv_sec: 3884437, tv_nsec: 462790315 }
loop process, 112.443µs
Polling message stream
msgstream found Ok(NotReady)
Sending status to server: [kPlayStatusPause]
==> kPlayStatusPause
scheduling Read for: 5
flushing framed transport
writing; remaining=772
framed transport flushed
loop poll - 376.163µs
loop time - Instant { tv_sec: 3884437, tv_nsec: 463706659 }
loop process, 80.499µs
event Readable | Writable Token(46137349)
loop process - 1 events, 0.000s
attempting to decode a frame
frame decoded from buffer
attempting to decode a frame
scheduling Read for: 5
flushing framed transport
framed transport flushed
loop poll - 27.293187ms
loop time - Instant { tv_sec: 3884437, tv_nsec: 491115790 }
loop process, 58.722µs
Polling message stream
msgstream found Ok(NotReady)
loop poll - 11.463µs
loop time - Instant { tv_sec: 3884437, tv_nsec: 491321529 }
loop process, 51µs

<!-- They have to be betwenn the `/summary` and the `/details` HTML tags, and inside the backticks '`' -->  
```

</details>

**Comment by Johannesd3** [comment]
Was fixed upstream, but not yet released: librespot-org/librespot#605

**Comment by georgefst** [comment]
Oh cool. In fact, this is a duplicate of #890 - not sure how I missed that one.
